cirq-google/cirq_google/serialization/op_deserializer.py

```diff
@@ -31,13 +31,12 @@ class OpDeserializer(abc.ABC):
 
     @property
     @abc.abstractmethod
-    def serialized_id(self) -> str:
-        """Returns the string identifier for the accepted serialized objects.
+    def can_deserialize_predicate(self) -> Callable[[Any], bool]:
+        """The method used to determine if this can deserialize a proto."""
```

Comment:
Can this be a regular method returning a bool instead?  Or a staticmethod if you want to have it decoupled from OpDeserializer object?

As it is, each call of `CircuitOpDeserializer.can_deserialize_proto` creates lambda object, calls it once, and discards it after the call.

(Same for `OpSerializer.can_serialize_predicate`.)

Reply:
Good idea.  I don't remember why this was done originally, but it is no longer needed.